RetroBar/Languages/русский.xaml

```diff
@@ -77,6 +90,13 @@
     <s:String x:Key="update_available">Доступно об_новление...</s:String>
     <s:String x:Key="exit_retrobar">Вы_йти из RetroBar</s:String>
     <s:String x:Key="customize_notifications_menu">Настройка _уведомлений...</s:String>
+    <s:String x:Key="lock_taskbar_menu">_Закрепить панель задач</s:String>
+
+    <s:String x:Key="show_desktop_tip_95">Показать рабочий стол</s:String>
```

Comment:
This line is duplicating the one below, despite having a little different meaning in original, which is okay, but maybe it would be better to come up with a more accurate one... 

Though, the only leaked build containing this exact line is English only, expectedly but unfortunately.

Reply:
These lines are used in the "Windows 95-98" theme and due to the lack of localization they always appear untranslated. It is better to leave this variant for now.